stats/test/app.test.js

```diff
@@ -371,11 +371,23 @@ describe('HTTP request handler', () => {
     })
   })
 
+  beforeEach(async () => {
+    await pgPools.stats.query('DELETE FROM daily_scheduled_rewards')
+    await pgPools.stats.query('DELETE FROM daily_reward_transfers')
+    await pgPools.stats.query('DELETE FROM participants')
+
+    // Insert participants used by the tests below
+    await pgPools.stats.query(`
+      INSERT INTO participants (id, participant_address)
+      VALUES (1, '0x20'), (2, '0x00')
+    `)
+  })
```

Comment:
We need to use `mapParticipantsToIds` instead of hard-coded participant ids.

Reply:
I recommend creating a helper that will ensure the address is mapped to an ID and then call the sql query to insert into `daily_scheduled_rewards`.